src/components/MultiSelectionCaseList.js

```diff
@@ -28,7 +28,7 @@ class MultiSelectionCaseList extends CaseList {
                     {
                         this.props.cases.map((item) =>
                             <tr className="case-item" key={item.id}>
-                                <td><input type="checkbox" value={item.id} checked={item.checked} onChange={this.toggleSelection.bind(this, item)} /></td>
+                                <td><input id={'select-'+item.id} type="checkbox" value={item.id} checked={item.checked} onChange={this.toggleSelection.bind(this, item)} /></td>
```

Comment:
oh yeah, react requires a key on elements that are iterated.  I thought it had to be key, but I guess not.

Reply:
doh, nm... this is the id that you were talking about that we use for the tests.  I see key above that